generator/sig_readme.tmpl

```diff
@@ -7,8 +7,12 @@
 * [{{.Day}}s at {{.UTC}} UTC]({{$.MeetingURL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.UTC}}&tz=UTC).
 {{- end }}
 
+{{ if .MeetingArchiveURL -}}
```

Comment:
are we requiring a MeetingArchiveURL (a link to notes) and is there any way to do data validation?  Most sigs seem to have it already? 

This doesn't block this PR or anything.  I'm just curious

Reply:
I think it's good practice to have it, but I don't think we're imposing it. Yes, all the sigs have it already. If however, for some reason it's missing, this prevents it from displaying a broken link.